query/src/storages/fuse/operations/append.rs

```diff
@@ -59,6 +60,8 @@ impl FuseTable {
         .await;
 
         let locs = self.meta_locations().clone();
+        let segment_info_cache = ctx.get_storage_cache_manager().get_table_segment_cache();
```

Comment:
What's the cache path?

Reply:
later, at line num 78 (of the append.rs), the path of the segment info cache item is given :
   ~~~
                        if let Some(ref cache) = segment_info_cache {
                            let cache = &mut cache.write().await;
                            cache.put(seg_loc, seg);
                        }
  ~~~